packages/components/image/src/image.vue

```diff
@@ -67,17 +71,18 @@ const emit = defineEmits(imageEmits)
 
 let prevOverflow = ''
 
+const { loading } = toRefs(props)
 const { t } = useLocale()
 const ns = useNamespace('image')
 const rawAttrs = useRawAttrs()
 const attrs = useAttrs()
 
+const imageSrc = ref('')
 const hasLoadError = ref(false)
-const loading = ref(true)
-const imgWidth = ref(0)
-const imgHeight = ref(0)
+const isLoading = ref(true)
 const showViewer = ref(false)
 const container = ref<HTMLElement>()
+const supportLoading = ref(true)
```

Comment:
Should we do something about this thing? Maybe detect the support of this API

Reply:
Tip `lazy` -> `loading="lazy"` ?

I do not recommend to do this. because native lazy loading will load more images than scroll (lazy API), and some older browsers do not support this feature.

Maybe we should not break the lazy API now.